docker-compose.dev.yaml

```diff
@@ -0,0 +1,8 @@
+version: '3.8'
```

Comment:
Wouldn't you want to also mount the code and run your app in `--reload` mode?

Reply:
What's the best way to pass arguments to the CMD directive? An `$RUN_FLAGS` Env var that will pass to unicorn?

Reply:
Probably best to juts repeat the command.
If you really want you can use an `ENTRYPOINT` directive instead, which is prepended to the container arguments, instead of replaced by them